apis/scheduling/v1alpha1/reservation_types.go

```diff
@@ -32,20 +32,26 @@ type ReservationSpec struct {
 	// like a normal pod.
 	// If the `template.spec.nodeName` is specified, the scheduler will not choose another node but reserve resources on
 	// the specified node.
+	// +optional
 	Template *corev1.PodTemplateSpec `json:"template,omitempty"`
 	// Specify the owners who can allocate the reserved resources.
-	// Multiple owner selectors and ANDed.
+	// Multiple owner selectors and ORed.
+	// +optional
 	Owners []ReservationOwner `json:"owners,omitempty"`
 	// By default, the resources requirements of reservation (specified in `template.spec`) is filtered by whether the
 	// node has sufficient free resources (i.e. ReservationRequest <  NodeFree).
 	// When `preAllocation` is set, the scheduler will skip this validation and allow overcommitment. The scheduled
 	// reservation would be waiting to be available until free resources are sufficient.
+	// +optional
 	PreAllocation bool `json:"preAllocation,omitempty"`
 	// Time-to-Live period for the reservation.
 	// `expires` and `ttl` are mutually exclusive. Defaults to 24h.
+	// +kubebuilder:default='24h'
```

Comment:
Hmm.. There is a scenario where some components don't want to set the expiration time, and the components that create the Reservation is responsible for destroying it. So is it reasonable to set a default time?

Reply:
So it means we can disable the expiration of the Reservation?
It's fine if we can manually clean up them.

Reply:
@hormes WDYT?

Reply:
As discussed with @hormes,  If `TTL` is nil, use the default value, else if `TTL` is 0, disable the expiration, if `TTL` > 0, use the value as expiration.